### PR TITLE
[MM-557] Added workflow file to add new issues to github board

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
- Added workflow file to add new issue to github board
https://github.com/orgs/mattermost/projects/9/views/1

### CODE REVIEW

**Review by mickmister — APPROVED** [review]

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:    Let's make sure @phoinixgrr reviews before merging.

File: .github/issue_add.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - uses: mattermost/github-app-installation-token-action@181cd1b8c94b158428c4facfe043d6e28c20be55
+        id: ghapp
+        with:
+          appId: "${{ vars.UNIFIED_CI_APP_ID }}"
+          installationId: "${{ vars.UNIFIED_CI_INSTALLATION_ID }}"
+          privateKey: ${{ secrets.UNIFIED_CI_PRIVATE_KEY }}
+      - uses: actions/add-to-project@31b3f3ccdc584546fc445612dec3f38ff5edb41c #v0.5.0
+        with:
+          project-url: https://github.com/orgs/mattermost/projects/9
+          github-token: ${{ steps.ghapp.outputs.token }}
```
Comment (toninis) [inline .github/issue_add.yml:26]: I do not want to share our private key to public repositories . I will create a PAT form this repo and change the code a bit .

**Review by toninis — CHANGES_REQUESTED** [review]

**Review by toninis — APPROVED** [review]

**Comment by wiggin77** [comment]
@raghavaggarwal2308 is this the only repo where this automation is needed?  If there are others, then there may be a better way to add this for all.

**Comment by raghavaggarwal2308** [comment]
@wiggin77 No we need this automation in all the major plugin so that we are aware of any new issues being created. On Michael's request I have already created a PR for Gitlab, Jira, Zoom. We can create the same PR for mscal, gcal and google drive plugin. 

Please let me know if there is a better way to do this. I am aware that there is separate repo for managing the CI across the plugin. Can we use that?

**Comment by wiggin77** [comment]
> No we need this automation in all the major plugin so that we are aware of any new issues being created. On Michael's request I hav

@toninis this automation will be needed for all supported plugins.  I feel like there might be a more centralized way to achieve this as you did for the `/update-branch`  commands.  Thoughts?

**Pull request merged**